packages/web/services/technology.js

```diff
@@ -359,3 +359,25 @@ export const createTechnologyReview = async (data) => {
 
 	return response.data;
 };
+
+/**
+ * Fetch current technology most recent comment.
+ *
+ * @param {string|number} id Technology id.
+ * @returns {object} The current technology most recent comment.
+ */
+export const getMostRecentComment = async (id) => {
+	if (!id) {
+		return {};
+	}
+
+	const response = await apiGet(`technologies/${id}/comments`);
+
+	if (response.status !== 200) {
+		return {};
+	}
+
+	const { data } = response;
+
+	return data[data.length - 1];
```

Comment:
you can use data.pop(); here 👍

Reply:
Yeah, but `pop` modifies the array. Is it something ok for us to do here?

Reply:
My 2 cents: we should avoid mutability 😄